app/assets/stylesheets/main.scss

```diff
@@ -82,6 +82,23 @@
   .grader_distribution {
     margin-bottom: 0;
   }
+
+  #grader-right {
+    height: 380px;
+    overflow: scroll;
+  }
+
+  ::-webkit-scrollbar {
+    -webkit-appearance: none;
```

Comment:
Yeah I don't know about this one. If @arkon knows what the correct solution is great, if not I'm fine ignoring it.